crates/gen/src/types/delegate.rs

```diff
@@ -140,6 +140,8 @@ impl Delegate {
                             ::std::ptr::null_mut()
                         };
 
+                    // TODO: implement IMarshal
```

Comment:
Nit: it would be nice to track these as issues instead of todos in the code.

Reply:
https://github.com/microsoft/windows-rs/issues/753